tests/jsx_escape/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -42,9 +42,9 @@ raw_amp = <span>foo & bar</span>
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 many_nbsp = <div>&nbsp; &nbsp; </div>;
 single_nbsp = <div>&nbsp;</div>;
-many_raw_nbsp = <div>&nbsp;&nbsp;&nbsp;</div>;
+many_raw_nbsp = <div> </div>;
 
-amp = <span>foo & bar</span>;
+amp = <span>foo &amp; bar</span>;
```

Comment:
Don't we want to keep the original value in this case https://github.com/prettier/prettier/blob/master/tests/jsx_escape/nbsp.js#L6?